addons/stock/wizard/stock_replenishment_info.py

```diff
@@ -122,13 +122,15 @@ def _compute_lead_time(self):
 
     @api.depends('warehouse_id', 'free_qty', 'uom', 'qty_to_order')
     def _compute_warning_message(self):
+        self.warning_message = ''
         for record in self:
-            self.warning_message = _('{0} can only provide {1} {2}, while the quantity to order is {3} {2}.').format(
-                record.warehouse_id.name,
-                record.free_qty,
-                record.uom,
-                record.qty_to_order
-            )
+            if self.free_qty < self.qty_to_order:
+                self.warning_message = _('{0} can only provide {1} {2}, while the quantity to order is {3} {2}.').format(
```

Comment:
Hmm it should be record I think

Reply:
I agree, I should have seen that the previous code was not correct either